src/test/java/teammates/test/cases/browsertests/InstructorSubmissionAdjustmentUiTest.java

```diff
@@ -39,8 +39,8 @@ protected void prepareTestData() {
         removeAndRestoreDataBundle(testData);
     }
 
-    @Test
-    public void testAdjustmentOfSubmissions() {
+
+    public void testAdjustmentOfSubsmissions() {
```

Comment:
This is an unrelated change that also happens to be causing the build failure (take a look at the build log by clicking on the red `x` next to your last commit).